eng/tox/tox.ini

```diff
@@ -79,6 +79,23 @@ commands =
     {envbindir}/python {toxinidir}/../../../eng/tox/run_mypy.py -t {toxinidir}
 
 
+[testenv:whl_no_aio]
+skipsdist = true
+skip_install = true
+changedir = {toxinidir}
+deps =
+  {[base]deps}
+commands =
+    - {envbindir}/python -m pip uninstall aiohttp --yes
+    {envbindir}/python {toxinidir}/../../../eng/tox/create_package_and_install.py -d {envtmpdir} -p {toxinidir} -w {envtmpdir}
+    {envbindir}/python -m pip freeze
+    pytest \
+      {[testenv]default_pytest_params} \
+      --ignore-glob='*_async.py' \
```

Comment:
Not having this means that async tests implode with 

![image](https://user-images.githubusercontent.com/45376673/113207351-5c9a6680-9225-11eb-853c-e85b0ec7caa2.png)